mysql-ballerina/build.gradle

```diff
@@ -433,28 +429,16 @@ updateTomlFile.dependsOn copyStdlibs
 startTestDockerContainer.dependsOn createTestDockerImage
 startTestSslDockerContainer.dependsOn createTestDockerImage
 
-ballerinaTest.dependsOn updateTomlFile
-ballerinaTest.dependsOn initializeVariables
-ballerinaTest.dependsOn ":${packageName}-native:build"
-ballerinaTest.dependsOn startTestDockerContainer
-ballerinaTest.dependsOn startTestSslDockerContainer
-ballerinaTest.finalizedBy revertTomlFile
-ballerinaTest.finalizedBy stopTestDockerContainer
-ballerinaTest.finalizedBy stopTestSslDockerContainer
-test.dependsOn ballerinaTest
-
-ballerinaBuild.dependsOn test
 ballerinaBuild.dependsOn updateTomlFile
 ballerinaBuild.dependsOn initializeVariables
 ballerinaBuild.dependsOn ":${packageName}-native:build"
 ballerinaBuild.finalizedBy revertTomlFile
 ballerinaBuild.finalizedBy stopTestDockerContainer
 ballerinaBuild.finalizedBy stopTestSslDockerContainer
+test.dependsOn startTestDockerContainer
+test.dependsOn startTestSslDockerContainer
+ballerinaBuild.dependsOn test
```

Comment:
I wonder what is the behavior of:
```
./gradlew clean build -x test
./gradlew clean test
```

Reply:
the command `./gradlew clean build -x test` is to build the project but ignore test cases. this is useful to check any compilation issues in the code without running test cases. on the other hand `./gradlew clean test` is useful when writing and testing test cases without any code changes.